### PR TITLE
Update AudioSetup.md

### PR DESCRIPTION
**Pull request by E3V3A**

### CODE REVIEW

File: docs/AudioSetup.md [hunk 1]
```diff
@@ -65,10 +65,10 @@ ctl.!default {
 
 Let's break this down:
 
-* in `pmc` the `p` stands for playback. So were configuring playback settings inside this block
+* in `pcm` stands for r `Pulse Code Modulation`.  Here we configure the playback settings inside this block
```
Comment (henrikra) [inline docs/AudioSetup.md]: What this lonely `r` means? :D
also could this `Here we configure the playback settings inside this block` be `Inside this block we configure the playback settings`?

File: docs/AudioSetup.md [hunk 2]
```diff
@@ -65,10 +65,10 @@ ctl.!default {
...
 * all you need to know that we are setting default card to 1 for playback
 * this number has to match your desired card that we looked up previous. For my case it is 1
-* in `ctl` the `c` stands for capture. So were configuring capture settings inside this block
+* in `ctl` stands for control and is setting the available controls for the device. Here we configure capture settings.
```
Comment (henrikra) [inline docs/AudioSetup.md]: Should control be like this: `control` like the `Pulse Code Modulation` above?

Comment (E3V3A) [reply docs/AudioSetup.md]: Hi @henrikra !
Sorry for such a late repply, I simply didn't see the notification for these comments. I'm part of so many repos, that I keep missing new notifications. Anyway..

* `r` is a typo! :)
* I believe that "control" refers to the available ALSA controls you find under `amixer -c 1`.
(I.e. you're using the controllers on card-1.)

**Comment by dolanmiu** [comment]
Is this ok to be merged in? @henrikra

**Comment by henrikra** [comment]
Well @E3V3A should fix those things pointed out in the comments first :P

**Comment by henrikra** [comment]
@E3V3A is dead

**Pull request closed without merging**